smh_app/settings.py

```diff
@@ -177,3 +221,93 @@
 SOCIAL_AUTH_SHAREMYHEALTH_HOST = os.environ.get('SMH_OAUTH_HOST')
 SOCIAL_AUTH_SHAREMYHEALTH_KEY = os.environ.get('SMH_OAUTH_KEY')
 SOCIAL_AUTH_SHAREMYHEALTH_SECRET = os.environ.get('SMH_OAUTH_SECRET')
+
+
+# Static files (CSS, JavaScript, Images)
+# https://docs.djangoproject.com/en/2.1/howto/static-files/
+STATICFILES_DIRS = [
+    os.path.join(BASE_DIR, 'sitestatic'),
+]
+
+STATIC_URL = '/static/'
+MEDIA_URL = '/media/'
+STATIC_ROOT = os.path.join(BASE_DIR, 'collectedstatic')
+MEDIA_ROOT = os.path.join(BASE_DIR, 'media')
+
+
+APPLICATION_TITLE = env('DJANGO_APPLICATION_TITLE',
+'Share My Health')
+
+LOGIN_REDIRECT_URL = 'home'
+
+EXTERNAL_AUTH_NAME = "Google"
+
+APPLICATION_TITLE = env('DJANGO_APPLICATION_TITLE',
+                        'Share My Health')
+ORGANIZATION_TITLE = env(
+    'DJANGO_ORGANIZATION_TITLE',
+    'Alliance for Better Health')
+ORGANIZATION_URI = env('DJANGO_ORGANIZATION_URI', 'https://abhealth.us')
+POLICY_URI = env(
+    'DJANGO_POLICY_URI',
+    'https://abhealth.us')
+POLICY_TITLE = env('DJANGO_POLICY_TITLE', 'Privacy Policy')
+TOS_URI = env('DJANGO_TOS_URI', 'https://abhealth.us')
+TOS_TITLE = env('DJANGO_TOS_TITLE', 'Terms of Service')
+TAG_LINE_1 = env('DJANGO_TAG_LINE_1', 'Share your health data')
```

Comment:
This can be changed to `TAG_LINE` and it's default value can have `\n` etc. to format.